ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestCasesRunner.scala

```diff
@@ -75,12 +75,17 @@ final class LedgerTestCasesRunner(
       session
         .createTestContext(testName, identifierSuffix)
         .flatMap { context =>
-          val start = System.nanoTime()
-          val result = test(context).map(_ => Duration.fromNanos(System.nanoTime() - start))
-          logger.info(
-            s"Started '${test.description}'${test.repetition.fold("")(r => s" (${r._1}/${r._2})")} with a timeout of $scaledTimeout."
-          )
-          result
+          val features = context.configuredParticipants.head.features
+          if (test.testCase.enabled(features)) {
```

Comment:
We currently manage other exclusions much higher up; is it possible to move this logic to the runner?

Reply:
Moved it to the highest level where possible. We have one higher level, but we don't have access to the features there.